### PR TITLE
Updated the onelive youtube API url

### LINKED ISSUE #1562: Update the onelive youtube API url
**Is your feature request related to a problem? Please describe.**
sef-dataholder is now deployed on dataholder.sefglobal.org. So, we need to update the related URLs.

**Describe the solution you'd like**
Update the youtube API Url here:
https://github.com/sef-global/sef-site/blob/e7a5ae486c9398f3392b5348f636c2adabd53f3d/onelive/functions.js#L53
https://github.com/sef-global/sef-site/blob/e7a5ae486c9398f3392b5348f636c2adabd53f3d/onelive/archive/functions.js#L5

The new URL is : `https://dataholder.sefglobal.org/youtube/onelive?maxResults=3`

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by gpsakthivel**
## Purpose

The purpose of this PR is to fix #1562

## Goals

- To update the onelive Youtube API URL.

### Preview Link

https://pr-1565-sef-site.surge.sh/onelive/

##  Checklist
- [x] I have read and understood the [development best practices](https://handbook.sefglobal.org/organisation/engineering-team#development-best-practices) guidelines
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by Piumal1999** [comment]
Good job @gpsakthivel 
Could you please update the pull request description according to the pull request template. (Example: https://github.com/sef-global/sef-site/pull/1564)

**Comment by gpsakthivel** [comment]
Sure @Piumal1999

File: onelive/archive/functions.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ let nextPageToken = null;
 let fetchedItemCount = 0;
 
 function loadYoutubeVideos() {
-    let url = "https://apps.sefglobal.org/youtube/onelive?maxResults=12";
+    let url = "https://dataholder.sefglobal.org/youtube/onelive?maxResults=3";
```
Comment (Piumal1999) [inline onelive/archive/functions.js]:
```suggestion
    let url = "https://dataholder.sefglobal.org/youtube/onelive?maxResults=12";
```

@gpsakthivel, here maxResults count should be 12

Comment (gpsakthivel) [reply onelive/archive/functions.js]: Resolved

**Review by Piumal1999 — APPROVED** [review]
Good job @gpsakthivel 
Thanks for contributing

**Pull request merged**